### PR TITLE
TT-345 update the documentation of circuit breaker

### PR DESCRIPTION
**Pull request by sredxny**
Updated the documentation of circuit breaker, mentioned how to disable the half-open state and fixed the type of circuit breaker that we use

### CODE REVIEW

**Review by marksou — APPROVED** [review]
👍

**Review by gregdelhon — APPROVED** [review]
LGTM

File: tyk-docs/content/planning-for-production/ensure-high-availability/circuit-breakers.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ weight: 3
...
...Tyk has a built-in circuit breaker pattern as a path-based option. Our circuit breaker is rate-based, so if a sample size `x` of `y%` requests fail, the breaker will trip. The Gateway will stop **all** inbound requests to that service for a pre-defined period of time (a recovery time-period). You configure this time period using the `return_to_service_after` option in your API definition, or setup via the Dashboard, the circuit breaker also has an state called half-open state that enable the ability to check if the problem is fixed, this is done by making real requests to the upstream before the time configured in  `return_to_service_after` happens, by default the tyk circuit breaker has enabled the half-open state, if the desired behavior is to only check after the time configured in `return_to_service_after` is consumed then you can disable this by setting  `disable_half_open_state` to true. See [Configure with the API Definition](#with-api) or [Configure with the Dashboard] (#with-dashboard). This also triggers an event which you can hook into to perform corrective or logging action. When a circuit breaker is tripped, it will return a 503 "Service temporarily unavailable" error.
```
Comment (letzya) [inline tyk-docs/content/planning-for-production/ensure-high-availability/circuit-breakers.md]: "that enable the ability" phrasing @marksou

**Review by letzya — CHANGES_REQUESTED** [review]
"that enable the ability" phrasing @marksou

File: tyk-docs/content/planning-for-production/ensure-high-availability/circuit-breakers.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/planning-for-production/ensure-high-availability/circuit-breakers.md]: Changed

**Pull request merged**